src/script/event/EventRepository.ts

```diff
@@ -131,21 +130,24 @@ export class EventRepository {
   // WebSocket handling
   //##############################################################################
 
-  private readonly updateConnectivitityStatus = (state: WEBSOCKET_STATE) => {
+  private readonly updateConnectivitityStatus = (state: ConnectionState) => {
     switch (state) {
-      case WEBSOCKET_STATE.CONNECTING: {
+      case ConnectionState.CONNECTING: {
         amplify.publish(WebAppEvents.WARNING.DISMISS, Warnings.TYPE.NO_INTERNET);
         amplify.publish(WebAppEvents.WARNING.SHOW, Warnings.TYPE.CONNECTIVITY_RECONNECT);
         return;
       }
-      case WEBSOCKET_STATE.CLOSING: {
+      case ConnectionState.PROCESSING_NOTIFICATIONS: {
+        amplify.publish(WebAppEvents.WARNING.DISMISS, Warnings.TYPE.NO_INTERNET);
+        amplify.publish(WebAppEvents.WARNING.DISMISS, Warnings.TYPE.CONNECTIVITY_RECONNECT);
+        amplify.publish(WebAppEvents.WARNING.SHOW, Warnings.TYPE.CONNECTIVITY_RECOVERY);
```

Comment:
Now the processing of notification stream gets a clean new lifecycle event. 
We can now handle `all` the banner displaying in this single method